core/src/cg/group4/util/audio/AudioPlayer.java

```diff
@@ -47,7 +66,9 @@ public AudioPlayer() {
         cPreferences = Gdx.app.getPreferences("AUDIO");
         cAudioEnabled = cPreferences.getBoolean("ENABLED", true);
         cDefaultMusic = Gdx.audio.newMusic(Gdx.files.internal("music/Summer Day.mp3"));
-//        cDefaultMusic = Assets.getInstance().getMusic("music/Summer Day.mp3");
+        cEventStartedSound = Assets.getInstance().getSound("sounds/eventOccured.mp3");
+        cTaskCompletedSound = Assets.getInstance().getSound("sounds/completedTask.wav");
+        cEventCompletedSound = Assets.getInstance().getSound("sounds/eventCompleted.mp3");
         cDefaultMusic.setLooping(true);
```

Comment:
I would say split the main music to a method instead of calling this already in the constructor. It is a different behavior than the initialization.